src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
 document.addEventListener('click', (e) => {
-  // write code here
+  const field = e.target.closest('.wall');
+
+  if (!field) {
+    return;
+  }
+
+  const wallRect = wall.getBoundingClientRect();
+  const spiderRect = spider.getBoundingClientRect();
+  const widthSpider = spider.offsetWidth;
+  const border = wall.clientTop;
+
+  let x = e.clientX - widthSpider / 2 - border - wallRect.left;
+  let y = e.clientY - widthSpider / 2 - border - wallRect.top;
```

Comment:
These calculations for 'x' and 'y' are a bit complex and could use some comments to explain what they're doing. It's important to comment your code to make it easier for others (and future you) to understand.